CHANGES.md

```diff
@@ -10,6 +10,7 @@
 
 - Fixed a bug where the scale of a `Model` was being incorrectly applied to its bounding sphere. [#10855](https://github.com/CesiumGS/cesium/pull/10855)
 - Fixed a bug where rendering a `Model` with image-based lighting while specular environment maps were unsupported caused a crash. [#10859](https://github.com/CesiumGS/cesium/pull/10859)
+- Fixed a bug where request render mode was broken when a ground primitive is added. [#10756](https://github.com/CesiumGS/cesium/issues/10756)
```

Comment:
This should be updated with the link to PR.

Reply:
We often link to the original issue fixed in `CHANGES.md` - [For instance](https://github.com/CesiumGS/cesium/blob/28ebe20d91b1715b349c5cef92f8a37f4405baf9/CHANGES.md?plain=1#L80)

Reply:
Oh, interesting. I never noticed that before, always thought it was only PRs.